lib/ringsc.gi

```diff
@@ -338,62 +338,61 @@ end );
 ##  where `$c_{ijk}$ = <sctable>[i][j][1][i_k]'
 ##  and `<sctable>[i][j][2][i_k] = k'.
 ##
-InstallGlobalFunction( RingByStructureConstants, function( arg )
+BindGlobal("DoRingBySC",function(marg,oneinfo)
```

Comment:
BTW you could retain the name `arg` instead of changing to `marg` - it only has a special meaning when it is the sole argument to the function.